src/main/java/seedu/duke/model/ItemContainer.java

```diff
@@ -4,6 +4,7 @@
 import seedu.duke.model.exception.IllegalArgumentException;
 import seedu.duke.model.exception.ItemNotExistException;
 
+import java.math.BigDecimal;
```

Comment:
No need this package alr cuz bigdecimal will be processed internally in Item class.